tests/shell/tests/01-run.py

```diff
@@ -74,7 +74,7 @@
     ('echo escaped\\ space', '"echo""escaped space"'),
     ('echo escape within \'\\s\\i\\n\\g\\l\\e\\q\\u\\o\\t\\e\'', '"echo""escape""within""singlequote"'),
     ('echo escape within "\\d\\o\\u\\b\\l\\e\\q\\u\\o\\t\\e"', '"echo""escape""within""doublequote"'),
-    ("""echo "t\e st" "\\"" '\\'' a\ b""", '"echo""te st"""""\'""a b"'),
+    ("""echo "t\e st" "\\"" '\\'' a\ b""", '"echo""te st"""""\'""a b"'),  # noqa: W605
```

Comment:
```suggestion
    ("echo \"t\e st\" \"\\"\" '\\'' a\ b", '"echo""te st"""""\'""a b"'),
```
... not sure this is easier to read. So maybe suppressing the warning is the better way to go :sweat_smile: